variables.tf

```diff
@@ -33,6 +50,11 @@ variable "install_log_forwarder" {
   type        = bool
   default     = false
   description = "Set to true to install the Datadog Log Forwarder (requires var.api_key to be set)"
+
+  validation {
+    condition     = !var.install_log_forwarder || var.create_api_key || try(length(var.api_key), 0) > 0
+    error_message = "The api_key value must be set or create_api_key has to be enabled if install_log_forwarder is set to true."
```

Comment:
Nice, love the validation!

cross variable validation means the minimum version needs to be bumped to v1.9.0> :-D